src/NodaTime/TimeZones/Resolvers.cs

```diff
@@ -67,6 +67,19 @@ public static class Resolvers
         };
 
         /// <summary>
+        /// A <see cref="SkippedTimeResolver"/> which shifts values in the "gap" forward by the duration
+        /// of the gap (which is usually 1 hour).
+        /// </summary>
+        public static readonly SkippedTimeResolver ReturnForwardShifted = (local, zone, before, after) =>
```

Comment:
Could you make this a read-only property (like ThrowWhenAmbiguous) and fix up the other fields while you're in the file? I'm trying to get rid of public fields from the API almost everywhere.